libs/ett/core/shell/src/lib/ett-shell.component.ts

```diff
@@ -35,19 +35,24 @@ const selector = 'ett-shell';
     `
       ${selector} {
         display: block;
+        width: 100%;
+        max-width: 1600px;
+        margin: 0 auto;
+        /** @todo: Agree on breakpoints - Full width and centered on screens larger than 1600px(?) */
       }
     `,
   ],
   template: `
     <watt-shell>
+      <!--
       <ng-container watt-shell-sidenav>
         <ett-primary-navigation></ett-primary-navigation>
       </ng-container>
 
       <ng-container watt-shell-toolbar>
         <h1>EnergyOrigin</h1>
       </ng-container>
-
+    -->
```

Comment:
question: Why is there a need to touch the shell component in this pull request?

Reply:
@LayZeeDK There's not :-) It was me not understanding if that shell components elements would be visible on "/" or not - I have commented it back in again.